assignment.js

```diff
@@ -113,6 +124,48 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+//GAME 1
```

Comment:
@Defy1x Nice! interesting use of nested if. However, this may not be as intuitive to people reading your code..an alternative approach would be to go with a "flatter" approach:

```
if (playerAChoice === 'rock' && playerBChoice === 'paper') {
 ...
} else if (playerAChoice === 'rock' && playerBChoice === 'scissors') { 
 ...
} ....
```